components/InformationTile.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+    <div class="rectangle-info"><p class="tile-content">{{ textContent }}</p></div>
+</template>
+
+<script lang="ts">
+export default {
+  name: 'RectangleInfo',
+  props: {
+    textContent: {
+      type: String,
+      required: true,
+    }
+  },
+};
+</script>
+
+<style>
+.rectangle-info {
+    width: 312px;
```

Comment:
I would make these a bit smaller or decrease margins. Ideally all three tiles should fit on a 13" laptop

![image](https://user-images.githubusercontent.com/67887553/212192590-7c6e0f2d-34f5-4abd-8b8d-ddeb80b7d971.png)